src/Pages/gunwookim/Login/Login.js

```diff
@@ -1,8 +1,66 @@
 import React from 'react';
 
+import { Link } from 'react-router-dom';
+
+import './Login.scss';
+
 class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      password: '',
+      isVerify: false,
+    };
+  }
+
+  handleInput = e => {
+    const { name, value } = e.target;
+    this.setState({ [name]: value }, this.loginValidation);
+  };
+
+  loginValidation = () => {
+    const { email, password } = this.state;
+    const emailVerify = email.includes('@');
+    const passwordVerify = password.length >= 5 ? true : false;
```

Comment:
```suggestion
    const isPasswordVerified = password.length >= 5;
```
- boolean 타입의 경우 `is-` prefix를 써주면 변수명만 보고도 타입을 추론하기 좋을 것 같습니다. 그리고 함수가 아니기 때문에 `-ed`로 끝내는 네이밍은 어떨지 제안드려봅니다.
- 평가된 계산식 자체가 boolean 값일 것이기 때문에 굳이 삼항연산자를 써주지 않아도 될 것 같습니다